src/core/Distribution.pm

```diff
@@ -57,7 +112,7 @@ class Distribution::Resources does Associative {
     }
 
     method AT-KEY($key) {
-        CompUnit::RepositoryRegistry.repository-for-spec($.repo).resource($.dist-id, $key)
+        CompUnit::RepositoryRegistry.repository-for-spec($.repo).resource($.dist-id, "resources/$key")
```

Comment:
Maybe, but the current implementation is a bug. Other dist objects are referred to by their relative path-name in `files` of META6. For instance: `resources/bin/xxx`, which would get stored as `bin/xxx` in `files`, would conflict with an actual script `bin/xxx` which also gets stored as `bin/xxx`